src/editor/CSSInlineEditor.js

```diff
@@ -162,7 +162,7 @@ define(function (require, exports, module) {
      *         {string} if pos is in tag but not in tag name, class attr, or id attr; or null if the
      *         selection isn't even close to a context where we could provide anything.
      */
-    function htmlToCSSProvider(hostEditor, pos) {
+    function htmlToCSSProvider(hostEditor, pos, type) {
```

Comment:
As soon as I see you making changes to a provider itself, I know that we have a problem, since it means that we're changing the provider API, and an arbitrary provider that you don't rewrite won't work.  6 months from now, I want to be able to plug in some new provider that we find, and not have to modify it at all to work with this code.  All your changes should be happening in `EditorManager` where these providers are registered and `Editor` where you listen for cursor activity.